nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestUpload.java

```diff
@@ -155,6 +155,27 @@ public void itCanAttachABatchToADoc() {
     }
 
     @Test
+    public void itCantAttachAnExistingBatchToADoc() {
+        //Upload file
+        BatchUpload batchUpload = nuxeoClient.fetchUploadManager();
+        String batchID = batchUpload.getBatchId();
+        assertNotNull(batchID);
+        File file = FileUtils.getResourceFileFromContext("sample.jpg");
+        batchUpload = batchUpload.upload(file.getName(), file.length(), "jpg", batchID, "1", file);
+        assertNotNull(batchUpload);
+
+        // Getting a doc and attaching the existing batch file
+        Document doc = new Document("file", "File");
+        doc.setPropertyValue("dc:title", "new title");
+        doc = nuxeoClient.repository().createDocumentByPath("/folder_1", doc);
+        assertNotNull(doc);
+        doc.setPropertyValue("file:content", batchUpload.getBatchBlob(batchID));
```

Comment:
For me it's useful for example:
```
@POST
public String initializeBatch(){
        BatchUpload batchUpload = nuxeoClient.fetchUploadManager();
        return batchUpload.getBatchId();
    }

@POST
@Path("{batchId}/execute")
public Document executeBatchUpload(@PathParam("batchId") String batchID, String json){
        Document document = new Document("file", "File");
        document.set("dc:title", "created by batchUpload");
        document = nuxeoClient.repository().createDocumentByPath(Const.FOLDER_PATH, document);
        document.set("file:content", nuxeoClient.fetchUploadManager().getBatchBlob(batchId));
        document = document.updateDocument();
        return document;
}
```
I need to create new blob in executeBatchUpload by **existing** batchUpload.
I hurried with pull request. My implementation is not working correctly now.